Atomic/syscontainers.py

```diff
@@ -172,7 +172,7 @@ def install_system_container(self, image, name):
             util.write_out("%s already present" % (name))
             return
 
-        return self._checkout_system_container(repo, name, image, 0, False)
+        return self._checkout_system_container(repo, name, image, 0, False, self.args.remote)
```

Comment:
Fixed the first one, but the update_system_container was just setting the remote=None so since that is the default, I don't think we need to change it.